CMakeLists.txt

```diff
@@ -1,12 +1,13 @@
-cmake_minimum_required(VERSION 3.5.0 FATAL_ERROR) #CPACK_DEBIAN_<COMPONENT>_PACKAGE_NAME
+cmake_minimum_required(VERSION 3.14...3.18 FATAL_ERROR) #CPACK_DEBIAN_<COMPONENT>_PACKAGE_NAME
 
 
-find_package(IRODS 4.3.0 EXACT REQUIRED)
+find_package(IRODS 5.0.0 REQUIRED)
 set(IRODS_PLUGIN_REVISION "0")
 
-set(CMAKE_C_COMPILER ${IRODS_EXTERNALS_FULLPATH_CLANG}/bin/clang)
-set(CMAKE_CXX_COMPILER ${IRODS_EXTERNALS_FULLPATH_CLANG}/bin/clang++)
-set(CMAKE_EXE_LINKER_FLAGS_INIT "${CMAKE_EXE_LINKER_FLAGS} -stdlib=libc++")
+include(RequireOutOfSourceBuild)
+include(IrodsCXXCompiler)
```

Comment:
Unsure if these are proper includes. Included since it corrected the clang version used for compilation.